sdk/appcenter-distribute/src/test/java/com/microsoft/appcenter/distribute/DistributeBeforeDownloadTest.java

```diff
@@ -87,11 +87,13 @@
 @PrepareForTest({
         DistributeUtils.class,
         SessionContext.class,
-        PermissionUtils.class,
-        Build.VERSION.class
+        PermissionUtils.class
 })
 public class DistributeBeforeDownloadTest extends AbstractDistributeTest {
 
+    @Rule
+    public Timeout mGlobalTimeout = Timeout.seconds(180);
```

Comment:
It's already defined in `AbstractDistributeTest`. 15s should be enough for testing. You can comment that rule during debugging